features/cart/shopping_cart/inability_of_adding_a_product_with_insufficient_stock.feature

```diff
@@ -0,0 +1,16 @@
+@shopping_cart
+Feature: Inability to add a specific product to the cart when it is out of stock
+    In order to buy only available products
+    As a Visitor
+    I want to be prevented from adding products which are not available in the inventory
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And the store has a product "T-shirt banana" priced at "€12.54"
+
+    @ui
+    Scenario: Not being able to add a product to the cart when it is out of stock
+        Given the product "T-shirt banana" is not available at the moment
```

Comment:
`is not available at the moment` this can be confusing because the product can be disabled, not available in the channel and probably few more, we should be more explicit e.g. `the product "X" is out of stock`